servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -1334,8 +1334,8 @@ public boolean mount(TachyonURI tachyonPath, TachyonURI ufsPath)
   }
 
   void mountFromEntry(AddMountPointEntry entry) throws InvalidPathException {
-    TachyonURI tachyonPath = entry.getTachyonPath();
-    TachyonURI ufsPath = entry.getUfsPath();
+    TachyonURI tachyonPath = entry.getTachyonURI();
```

Comment:
Since the data type is called TachyonURI, I would keep the method name as is for consistency.